kerckhoff/packages/serializers.py

```diff
@@ -172,3 +174,82 @@ def get_version_data(self, obj: Package):
     class Meta(PackageSerializer.Meta):
         fields = PackageSerializer.Meta.fields + ("version_data",)
         read_only_fields = PackageSerializer.Meta.read_only_fields + ("version_data",)
+
+class PackageInfoSerializer(serializers.ModelSerializer):
+    package_set = serializers.StringRelatedField()
+    latest_version = serializers.StringRelatedField()
+    description = serializers.SerializerMethodField()
+    folder_id = serializers.SerializerMethodField()
+    folder_url = serializers.SerializerMethodField()
+    metadata = serializers.SerializerMethodField()
+    data = serializers.SerializerMethodField()
+    cached_article_preview = serializers.SerializerMethodField()
+
+    def get_description(self, obj: Package):
+        obj.fetch_cache()
+        versionSerializer = PackageVersionSerializer(obj.get_version(obj.latest_version.id_num))
```

Comment:
Please add some error handling in case no versions have been created yet. Right now it will just crash the server.

Reply:
Actually this serializer should be able to return something for packages even if they have no version, because the list packages endpoint should show every package even if no version has been created

Reply:
Done.